salt/_modules/metalk8s_kubernetes.py

```diff
@@ -2373,3 +2405,139 @@ def node_uncordon(node_name, **kwargs):
         salt '*' kubernetes.node_uncordon  node_name="minikube"
     '''
     return node_set_unschedulable(node_name, False, **kwargs)
+
+
+def get_crd_plural_from_kind(kind, **kwargs):
+    plural = None
+
+    crds = customresourcedefinition(**kwargs)
+
+    for crd in crds.get('items', []):
+        crd_names = crd.get('spec', {}).get('names', {})
+        if kind == crd_names.get('kind'):
+            plural = crd_names.get('plural')
```

Comment:
You could have simply done something like:

```suggestion
            return crd_names.get('plural')
```

And get rid of the `break` statement and the `plural` variable

Reply:
Yeah right, at the beginning it was part of another function, I forgot to change it when I moved this.
I will change that.